ci/prow/e2e_prebuilt_kernel_module

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Label the node so that it gets the module..."
+kubectl label node minikube wants-module=kmm_ci_a
```

Comment:
Do we really need to label the node? Can we just tell it to run on all workers/masters?

Here is a selector example that does that:
```
  selector:
    node-role.kubernetes.io/worker:
```